ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -37,6 +37,13 @@ if [[ ${ZONES_COUNT} -gt ${MAX_ZONES_COUNT} ]]; then
   exit 1
 fi
 
+MASTER_REPLICAS=3
+if [[ "${SIZE_VARIANT}" == "compact" ]]; then 
```

Comment:
In the next iteration, it would be handy if the script failed for every invalid value.